### PR TITLE
Remove identity verification from help content

### PR DESCRIPTION
**Pull request by michelle-rago**
## Changes proposed in this pull request:
- Removed info about identity verification

We will add info about verification needed for new requests and info about work email vs. personal email before the next launch.

### CODE REVIEW

**Comment by michelle-rago** [comment]
[Preview these changes](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/getgov-home/remove-identity-verification/help/account-management/)

File: pages/help/help_account-management.md [hunk 1]
```diff
@@ -23,26 +23,17 @@ In November 2023, we launched a new .gov registrar. You must have a Login.gov ac
 
 ### If you had an account in the old .gov registrar
 
-If you had a .gov account prior to November 2023, you can’t use your old credentials to access the new registrar. You’ll need to create a Login.gov account and provide the same email address you used to log in to the old registrar. That will ensure your domains are linked to your Login.gov account. 
+If you had a .gov account prior to November 2023, you can’t use your old credentials to access the new registrar. You’ll need to create a Login.gov account with the same email address you used to log in to the old registrar. That will ensure that you have access to your domains. 
+
 If you need help finding the email address you used in the past, [contact us](../../contact/).
 
-## Get help with your Login.gov account
+## Get help with Login.gov
```
Comment (neilmb) [inline pages/help/help_account-management.md:30]: **comment (non-blocking)**: Changing this section header will change the ID of the internal link that needs to be used in the navigation (see above).

File: pages/help/help_account-management.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ eleventyNavigation:
 ---
  
 - [Access the .gov registrar](#access-the-.gov-registrar)
-- [Get help with your Login.gov account](#get-help-with-your-login.gov-account)
+- [Get help with Login.gov](#get-help-with-your-login.gov-account)
```
Comment (neilmb) [inline pages/help/help_account-management.md:14]: **comment (non-blocking)**: changing the section header text below means needing to change the target of this link

```suggestion
- [Get help with Login.gov](#get-help-with-login.gov)
```

**Review by neilmb — APPROVED** [review]
One link change, but otherwise this looks good.

**Comment by michelle-rago** [comment]
> One link change, but otherwise this looks good.

Thank you!!

**Review by Katherine-Osos — APPROVED** [review]

**Pull request merged**